stdlib/transactions/src/main/ballerina/transactions/commons.bal

```diff
@@ -54,20 +53,24 @@ function scheduleTimer (int delay, int interval) returns boolean {
 function cleanupTransactions() returns error? {
     worker w1 {
         foreach _, twopcTxn in participatedTransactions {
-            string participatedTxnId = getParticipatedTransactionId(twopcTxn.transactionId, twopcTxn.transactionBlockId);
+            string participatedTxnId = getParticipatedTransactionId(twopcTxn.transactionId,
+                twopcTxn.transactionBlockId);
             if (time:currentTime().time - twopcTxn.createdTime >= 120000) {
                 if (twopcTxn.state != TXN_STATE_ABORTED && twopcTxn.state != TXN_STATE_COMMITTED) {
                     if (twopcTxn.state != TXN_STATE_PREPARED) {
-                        boolean prepareSuccessful = prepareResourceManagers(twopcTxn.transactionId, twopcTxn.transactionBlockId);
+                        boolean prepareSuccessful = prepareResourceManagers(twopcTxn.transactionId,
```

Comment:
I would rather break it like this:

boolean prepareSuccessful = 
...............................prepareResourceManagers(twopcTxn.transactionId, twopcTxn.transactionBlockId);